examples/va/nfv/ovs-dpdk/README.md

```diff
@@ -0,0 +1,30 @@
+# OpenStack OvS DPDK
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/12c57baeca4ae33dd30a7707d330eb094309b4cd) on Dec 6th, 2023**
```

Comment:
Is this true or just a copy-paste from a previous VA?

Reply:
@abays yes just copied. updated date based on my PR's.

> Is this true or just a copy-paste from a previous VA?

@abays yes just copied. updated date based on my PR's.

Reply:
> Is this true or just a copy-paste from a previous VA?

done